packages/calcite-components/src/components/tile/tile.tsx

```diff
@@ -29,6 +46,8 @@ export class Tile implements InteractiveComponent {
 
   /**
    * When `true`, the component is active.
+   *
+   * @deprecated
```

Comment:
Doc: Should this recommend using `tile-group` + a selection mode? cc @geospatialem @DitwanP 

This can be done in a follow-up PR.